doc/source/methods/ALE.ipynb

```diff
@@ -0,0 +1,285 @@
+{
```

Comment:
<ul><li>_a default argument, <code>min_bin_points = 4</code>, -&gt; commas are missing</li><li>I feel we could spell _data related_ as <em>data-related</em></li><li>random comment: what is your opinion on when having <code>min_bin_points</code> actually hurts us? Can we think of an example of a feature where this is not so big of a deal?</li><li>what exactly do those values in <code>ale0</code> represent. A quick explanation (if possible) or reference to somewhere where I can get an explanation would be helpful.</li><li>The construction _one for each feature_ is both parenthesized and used as an incident sentence - we can be consistent?</li><li>Not sure what ALE_iris is doing there? I think that's an img that's not rendering ...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<ul><li><code>min_bin_points</code>  can be quite sensitive, but would be a case-by-case basis, hard to come up with a general example</li><li>Reworded: "<code>ale0</code> - a list of "centering" values, one for each feature, used by the algorithm to center the <code>ale_values</code> (i.e. the sum of <code>ale_values</code> and <code>ale0</code> will be the uncentered ALE."</li><li> ALE_iris renders fine for me both on the notebook and the static docs, will need to check with you</li><li>All other small points addressed</li></ul>